torch/_inductor/utils.py

```diff
@@ -558,6 +534,8 @@ def has_uses(
     target: Node,
     use_selector_fn: Callable[[torch._ops.OpOverload], bool] = lambda _: False,
     use_aggregate_type: LogicalConnective = LogicalConnective.OR,
+    *,
+    select_output: bool = False,
```

Comment:
Instead of adding another arg here, can you just make `use_selector_fn` return True on output?